lib/chai/core/assertions.js

```diff
@@ -2145,4 +2145,27 @@ module.exports = function (chai, _) {
       , 'expected #{this} to not be frozen'
     );
   });
+
+  /**
+   * ### .finite
+   *
+   * Asserts that the target is a finite number. Unlike `.a('number')`, this will fail for `NaN` and `Infinity`
+   *
+   *     expect(foo).to.be.finite();
```

Comment:
Maybe we could change `foo` for any other number and add an example using `.not` and `NaN` or `Infinity`.